client/js/verification-portal/index.js

```diff
@@ -15,6 +15,7 @@ const mapDispatchToProps = dispatch => {
 			dispatch(getResourceToVerify)
 		},
 		changeFieldValue: (field, value) => {
+			value? value = value : value = ""
```

Comment:
@bhavyaab Yay, this is awesome and such a simple change! 👏 

Just a question on code readability. What do you think of this way of writing this line instead?
```
const changedValue = value ? value : ""
dispatch(change('verificationPortal', field, value))
```